test/testcafe/spec/TerminalView_spec.js

```diff
@@ -61,10 +62,10 @@ const terminalUnlockAccountFailedPermissionsMock = RequestMock()
   .onRequestTo('http://localhost:3000/idp/idx/introspect')
   .respond(terminalUnlockAccountFailedPermissions);
 
-fixture('Terminal view');
+fixture('Terminal view').meta('v3', true);
 
 async function setup(t) {
-  const terminalPageObject = new TerminalPageObject(t);
+  const terminalPageObject = process.env.OKTA_SIW_V3 ? new TerminalPageObjectV3(t) : new TerminalPageObject(t);
```

Comment:
Sorry this is a bit of a late comment - we can update separately so as not to block this PR since I only just worked out the new approach:

We are trying to put any ternaries that choose between v3 and v2 selectors as low and close to the selector itself as possible. This means putting a ternary in the PageObject itself, rather than switching between different PageObjects.